.github/workflows/run_test_cases_pr_comment.yml

```diff
@@ -22,12 +22,15 @@ jobs:
           gh run download -R "${{ github.repository }}" --name "pr.txt" "$RUN_ID"
           gh run download -R "${{ github.repository }}" --name "comment.txt" "$RUN_ID"
           gh run download -R "${{ github.repository }}" --name "commentwin.txt" "$RUN_ID"
+          [ -L pr.txt ] || [ -L comment.txt ] || [ -L commentwin.txt ] && exit 1
```

Comment:
Hi, @RedYetiDev.
May I ask why did you add this check and exit with error code 1 here ?
It will result in the error of downloading artifacts. https://github.com/cocos/cocos-engine/actions/runs/9828213649/job/27131836889

Reply:
Hey, that check verifies none of the files are symlinks, I’m not sure why it would exit 1 unless a symlink was uploaded?

Reply:
In fact, there are no symlink files in the artifacts.

You could try download the artifacts here: https://github.com/cocos/cocos-engine/actions/runs/9827700723
and unzip them.

And, I have just refactored run_test_cases_pr_comment.yml a little bit.  I used `actions/download-artifact@v4` instead of `gh run download`.

https://github.com/cocos/cocos-engine/pull/17327

Reply:
Thanks! I’ll also take a crack at it tomorrow